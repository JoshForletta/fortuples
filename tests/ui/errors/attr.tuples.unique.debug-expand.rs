use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::debug_expand]
    #[tuples::debug_expand(path = "")]
    impl Container<#Tuple> {}
}

fn main() {}
