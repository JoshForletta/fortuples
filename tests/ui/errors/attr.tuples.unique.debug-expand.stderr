error: `debug_expand` should be set only once
 --> tests/ui/errors/attr.tuples.unique.debug-expand.rs:7:15
  |
7 |     #[tuples::debug_expand(path = "")]
  |               ^^^^^^^^^^^^

error: the first definition was here
 --> tests/ui/errors/attr.tuples.unique.debug-expand.rs:6:15
  |
6 |     #[tuples::debug_expand]
  |               ^^^^^^^^^^^^
